### PR TITLE
[BACKLOG-39772] type column is not getting populated properly

### PR DESCRIPTION
**Pull request by srallapa**

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

File: ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java [hunk 1]
```diff
@@ -125,7 +137,11 @@ public final String getOutputPath() {
     String token = "outputFile = ";
     int index = resource.indexOf( token );
     if ( index < 0 ) {
-      return "";
+      token = "output file=";
```
Comment (peterrinehart) [inline ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java:140]: Should this match line 137 above "outputFile = " for consistency?  This string is looked for specifically elsewhere, so it shouldn't vary.

File: ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPanel.java [hunk 2]
```diff
@@ -519,7 +519,7 @@ public String getValue( JsJob job ) {
     // table.addColumn(idColumn, "ID");
     table.addColumn( nameColumn, Messages.getString( "scheduleName" ) );
     table.addColumn( scheduleColumn, Messages.getString( "recurrence" ) );
-    table.addColumn(executableType, Messages.getString( "executableType" ) );
+    table.addColumn(type, Messages.getString( "Type" ) );
```
Comment (peterrinehart) [inline ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPanel.java:522]: Is this change necessary?  Doesn't do any harm, but curious why it's needed, especially if we're changing the column heading in the UI.

**Review by peterrinehart — CHANGES_REQUESTED** [review]

File: ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java (hunk 1 above)
Comment (srallapa) [reply ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java:140]: I see in some cases its giving the string as following examples, so I needed to use both the tokens to see and take the output path.
input file = /home/admin/footrans.ktr:output file=/home/admin/footrans*
input file = /home/admin/footrans.ktr:outputFile = /home/admin/footrans-puc-6.*

File: ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPanel.java (hunk 2 above)
Comment (srallapa) [reply ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPanel.java:522]: We are changing the column header to match with PDI, in PDI its showing as Type, so to match in PUC I have updated.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 11m 26s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **120**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pentaho-scheduler-plugin/job/PR-88/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 12m 4s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **120**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pentaho-scheduler-plugin/job/PR-88/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java (hunk 1 above)
Comment (dcleao) [reply ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java]: If this is the case, then there would be other bugs awaiting to be fixed. In this same file, other places test for `:outputFile = /`.

References to `output file=`: https://github.com/search?q=org%3Apentaho+%22output+file%3D%22&type=code

References to `:outputFile =`: https://github.com/search?q=org%3Apentaho+%22%3AoutputFile+%3D%22&type=code

This does not seem right. Should a spike be created to investigate?

Comment (srallapa) [reply ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java]: > If this is the case, then there would be other bugs awaiting to be fixed. In this same file, other places test for `:outputFile = /`.
> 
> References to `output file=`: https://github.com/search?q=org%3Apentaho+%22output+file%3D%22&type=code
> 
> References to `:outputFile =`: https://github.com/search?q=org%3Apentaho+%22%3AoutputFile+%3D%22&type=code
> 
> Possibly related new issue: https://hv-eng.atlassian.net/browse/BACKLOG-39834.
> 
> This does not seem right. Should a spike be created to investigate?

The code is been updated in a way that it dont look for the reference like either "output file=: " or "outputFile =:" or any other, it takes the output path which is been separated with : after the input path, and then it takes content after the "=" part and trim, so it gets correct path.
